CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased for Desktop]
 
+### Fixed
+- Stream Key is now hidden. ([#7127](https://github.com/lbryio/lbry-desktop/pull/7127))
+
+## [0.51.3] - [2021-9-18]
+
```

Comment:
Please undo this.  You'll just need to add `Stream Key is now hidden. ([#7127](https://github.com/lbryio/lbry-desktop/pull/7127))` into the existing `### Fixed` entry under `## [Unreleased for Desktop]`.  No need to bump the version.

Reply:
Undone and placed in correct area.